### PR TITLE
[Helix] Crash/hang prototype

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

File: eng/helix/content/RunTests/TestRunner.cs [hunk 1]
```diff
@@ -247,7 +247,8 @@ public async Task<int> RunTestsAsync()
             {
                 // Timeout test run 5 minutes before the Helix job would timeout
                 var cts = new CancellationTokenSource(Options.Timeout.Subtract(TimeSpan.FromMinutes(5)));
-                var commonTestArgs = $"test {Options.Target} --logger:xunit --logger:\"console;verbosity=normal\" --blame \"CollectHangDump;TestTimeout=5m\"";                
+                var diagLog = Path.Combine(Environment.GetEnvironmentVariable("HELIX_WORKITEM_UPLOAD_ROOT"), "log.txt");
```
Comment (HaoK) [inline eng/helix/content/RunTests/TestRunner.cs:250]: should rename to vstest.log

**Comment by danmosemsft** [comment]
@HaoK would you mind resolving merge conflicts so we can try this again?

**Comment by HaoK** [comment]
Ah thanks for noticing @danmosemsft !  Should produce some new results now

**Comment by danmosemsft** [comment]
and again?

**Pull request closed without merging**